### ISSUE TITLE
Entry form with datetime-local field can't be saved bug

### ISSUE THREAD

**Issue by siebird** [craftcms/cms#5826]
I have an entry form on the frontend that fails to save when using `datetime-local` input. If I switch the field to a `Plain Text` field the entry saves, however, the timezone isn't respected.

Looking at how the `Date/Time` fields are stored when saving in the CP the format is:

- `2020-03-21 11:15`

On the frontend, the `datetime-local` `POST` variable is formatted like:

- `2020-03-21T11:15`

Does Craft strip the `T` and add a space when creating/editing within the CP? 

- Craft version: 3.4.9

**Comment by siebird** [comment]
Still troubleshooting through this. I upgraded Craft to 3.4.10.1. 

A couple odd things going on:

1. Switching the entry form fields to plain text and removing the `hidden` timezone field is working
2. However, no success/error response back in my AJAX call. Returns `undefined` in console.
3. The time is modified -4 hours since the timezone field was removed
4. Switching to `datetime-local` and adding the `hidden` timezone field back in throws `Couldn't save error` once you refresh the entries page in the CP.
5. Keeping `datetime-local` field and removing timezone field is working, however, time is modified -4 hours like in plain text scenario
6. Still no responses back from Craft on success

Seems `<input type="hidden" name="fields[timeStart][timezone]" value="America/New_York">` is the culprit.
